README.md

```diff
@@ -84,7 +84,7 @@ If there were any failures, run the tests manually with the `/debugtests` flag u
 
 1. `$(RepoRoot)\artifacts\test\$(Configuration)\$(Platform)\Test\RunDrts.cmd /name=DrtXaml /debugtests`
 2. Enter following command into the cmd window that pops up:
-`"%ProgramFiles%\Microsoft Visual Studio\2019\Preview\Common7\IDE\devenv.exe" DrtXaml.exe`
```

Comment:
I guess we should leave this to Preview.

Reply:
Done